packages/wdio-utils/src/test-framework/testFnWrapper.ts

```diff
@@ -7,6 +7,18 @@ import type {
     BeforeHookParam,
     AfterHookParam
 } from './types.js'
+import { afterHookErrMessage } from '../constants.js'
+
+declare global {
+    // Firstly variable '_wdioDynamicJasmineResult' gets value in packages/wdio-jasmine-framework/src/index.ts and then used here in wdio-utils/ as workaround for Jasmine
+    // eslint-disable-next-line no-var
+    var _wdioDynamicJasmineResult: WdioDynamicJasmineResult
+}
+
+interface WdioDynamicJasmineResult {
+    passedExpectations: [any, ...any[]]
+    failedExpectations: [any, ...any[]]
+}
```

Comment:
```suggestion
declare global {
    // Firstly variable '_wdioDynamicJasmineResult' gets value in packages/wdio-jasmine-framework/src/index.ts and then used here in wdio-utils/ as workaround for Jasmine
    // eslint-disable-next-line no-var
    var _wdioDynamicJasmineResult: Error
}
```

Reply:
Let's store only the error in the global variable since it is the only think we are interested in.

Reply:
Done

Reply:
Here I left type `any` instead of `Error `due to type-checking errors during transpilation